photon-lib/src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -106,7 +109,7 @@ public static void setVersionCheckEnabled(boolean enabled) {
      */
     public PhotonCamera(NetworkTableInstance instance, String cameraName) {
```

Comment:
Fix for the below: add a MultiSubscriber member, and add this to the constructor:

```java
        multiSubscriber = new MultiSubscriber(instance, 
            new String[]{"/photonvision/"},
            new PubSubOption[]{PubSubOption.topicsOnly(true)});
```